pydm/tests/conftest.py

```diff
@@ -102,6 +102,7 @@ class ConnectionSignals(QObject):
     prec_signal = pyqtSignal(int)
     upper_ctrl_limit_signal = pyqtSignal([float])
     lower_ctrl_limit_signal = pyqtSignal([float])
+    waveform_signal = pyqtSignal([int, int])
```

Comment:
Would you mind to put a comment on this signal so we know that this one is used just at the Waveform Table?